mantle/cmd/kola/options.go

```diff
@@ -125,7 +125,7 @@ func init() {
 	sv(&kola.GCPOptions.ServiceAcct, "gcp-service-account", "", "GCP service account to attach to instance (default project default)")
 	bv(&kola.GCPOptions.ServiceAuth, "gcp-service-auth", false, "for non-interactive auth when running within GCP")
 	sv(&kola.GCPOptions.JSONKeyFile, "gcp-json-key", "", "use a service account's JSON key for authentication (default \"~/"+auth.GCPConfigPath+"\")")
-	bv(&kola.GCPOptions.Confidential, "gcp-confidential-vm", false, "create confidential instances")
+	sv(&kola.GCPOptions.ConfidentialType, "gcp-confidential-type", "", "create confidential instances: sev, sev_snp, sev-snp")
```

Comment:
is there a reason for us to support both `sev_snp` and `sev-snp`? I'd say let's just accept one of them to make things cleaner (i.e. no need for `string.Replace`). 

If we must support both then I'd recommend not mentioning both here and silently support it in the code (i.e. with the existing `string.Replace` that you have in the code).

Reply:
Agree that will make more confusion, remove `sev-snp` in the help and silently support it in the code. Thanks!